doc/user/content/transform-data/troubleshooting.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Troubleshooting"
+description: ""
+menu:
+  main:
+    name: "How to troubleshoot common scenarios where Materialize is not working as expected."
+    identifier: troubleshooting-transform
+    parent: transform
+    weight: 25
+---
+
+## Why is my query slow?
+### Overview
+<!-- TODO: update this to use the query history UI once it's available -->
+Your query may be taking longer than expected for a handful of reasons, including:
+* Processing lag in upstream dependencies, like materialized views and indexes
+* Index design
+* Query design
+
+Below we detail how you can detect and address each of these issues and more.
+
+### Lagging materialized views or indexes
+#### Detect
+When a materialized view or index upstream of your query is behind on processing, your query must wait for it to catch up before returning. This is how we ensure consistent results for all queries.
+
+To see whether any materialized views or indexes are lagging you can look at the workflow graphs for your objects in the Materialize console.
+1. Go to https://console.materialize.com/
+2. Click on the "Clusters" tab in the nav bar
+3. Click on the cluster in which your upstream materialized view or index resides
+4. Go to the applicable "Materialized Views" or "Indexes" section
+5. Click on the materialized view or index's name
+
+If you find that one of the upstream materialized views or indexes is lagging, this could be the cause of your query slowness.
+
+#### Address
+To troubleshoot and fix a lagging materialized view or index, follow the steps in the [dataflow troubleshooting](/transform-data/dataflow-troubleshooting) guide.
+
+*Do you have multiple materialized views chained on top of each other? Are you seeing small amounts of lag?*\
```

Comment:
```suggestion
*Do you have multiple materialized views chained on top of each other? Are you seeing small amounts of lag?*
```
Extra `\`?

Reply:
That's markdown syntax for a newline without a new paragraph.
Without that it would wrap the "Tip:..." text into the previous line.

@morsapaes is there a more proper way to do that in markdown?

Reply:
I typically use `<br/>`, but `\` works too!